docs/samples/v1/tasks_nonfunctional_example.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: tekton.dev/v1
```

Comment:
What's this example supposed to illustrate if it's nonfunctional? I'm worried this will be confusing because it looks pretty functional